### ISSUE TITLE
nsxt_manager_status does not fully check of all manager components are online

### ISSUE THREAD

**Issue by laidbackware** [vmware/ansible-for-nsxt#305]
With  NSX-T 3.0.1 the nsxt_manager_status task completes successfully, but even after a 60 second delay I insert the next task fails because the manager is not fully online. The module should look in the contents of the response, as the status endpoint can be online when not all services are online.

```
# 11:41:50
ok: [127.0.0.1] => {
    "changed": false,
    "invocation": {
        "module_args": {
            "hostname": "192.168.0.193",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "username": "admin",
            "validate_certs": false,
            "wait_time": 50
        }
    },
    "msg": " NSX manager is UP"
}

# 11:45:25
ok: [127.0.0.1] => {
    "changed": false,
    "delta": 60,
    "echo": true,
    "rc": 0,
    "start": "2020-07-29 10:45:25.679587",
    "stderr": "",
    "stdout": "Paused for 60.0 seconds",
    "stop": "2020-07-29 10:46:25.679760",
    "user_input": ""
}

# 11:46:25
The full traceback is:
  File "/tmp/ansible_nsxt_licenses_payload_3i0plts8/ansible_nsxt_licenses_payload.zip/ansible/modules/nsxt_licenses.py", line 124, in main
  File "/tmp/ansible_nsxt_licenses_payload_3i0plts8/ansible_nsxt_licenses_payload.zip/ansible/module_utils/vmware_nsxt.py", line 73, in request
    raise Exception (data['error_code'], data)
fatal: [127.0.0.1]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "hostname": "#######",
            "license_key": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "state": "present",
            "username": "admin",
            "validate_certs": false
        }
    },
    "msg": "Failed to add license. Request body [{\"license_key\": \"********\"}]. Error[(101, {'module_name': 'common-services', 'error_message': 'Some appliance components are not functioning properly.', 'details': 'Component health: MANAGER:UNKNOWN, POLICY:UNKNOWN, SEARCH:UNKNOWN, UI:UP, NODE_MGMT:UP', 'error_code': 101})]."
}
```

**Comment by laidbackware** [comment]
Here is a workaround using the Ansible uri module:

```
    - name: Check all services are up
      uri:
        url: https://{{ nsx_manager.hostname }}/api/v1/cluster-manager/status
        user: "{{ nsx_manager.username }}"
        password: "{{ nsx_manager.password }}"
        method: GET
        force_basic_auth: yes
        validate_certs: False
        return_content: yes
      retries: 60
      delay: 10
      register: result
      until: "'UNKNOWN' not in result.content and 'DOWN' not in result.content"
```

**Comment by Akhilesh2412** [comment]
Fixed thiss issue in #316
